components/src/core/Hero/Hero.tsx

```diff
@@ -65,14 +66,34 @@ export type HeroProps = {
     iconSize?: number;
     label: string;
     onClick?: Function;
+    showTooltipOnHover?: boolean;
+    tooltipClasses?: object;
```

Comment:
We should do custom classes through the `classes` prop, not a prop for each class